src/components/Book.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Book = () => (
+  <>
+    <h1 className="newbook">New book is being added soon stay tuned!</h1>
+    {/* <h4>Book Author</h4> */}
```

Comment:
- [ ] To keep your code clean. Please remove all unused commented codes from your project.